tithiwa/group.py

```diff
@@ -142,22 +137,30 @@ def _find_group(self, groupname):
             name = curractive.find_element(By.CSS_SELECTOR, SELECTORS.GROUPS.CONTACTS_SEARCH_NAME).get_attribute(
                 'innerText')
             if name == groupname:
-                group_found = True
+                isfound = True
                 break
             preactive = curractive
-        if not group_found:
-            return None
-        else:
-            return curractive
+        self._wait_for_group_to_open(groupname)
+        return isfound
+
+    def _wait_for_group_to_open(self, groupname):
```

Comment:
2. Keep in mind every task takes time to be done and we need to find a way to detect things to be done, even the things seems instant